assets/wizards/engagement/views/newsletters/index.js

```diff
@@ -120,20 +124,115 @@ export const NewspackNewsletters = ( { className, onUpdate, isOnboarding = true
 	);
 };
 
-const Newsletters = () => {
-	const [ { status, url, error, newslettersConfig }, updateConfiguration ] = hooks.useObjectState(
-		{}
+export const SubscriptionLists = ( { onUpdate } ) => {
```

Comment:
`onUpdate` prop is not used in any instance, should it be included here?

Reply:
It's not currently used but I don't see why not keep it as a component feature.